src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -2891,7 +2972,7 @@ export class ResultsViewPageStore {
         invoke: async () => pickCopyNumberEnrichmentProfiles(this.molecularProfilesInStudies.result!)
     });
 
-    readonly copyNumberHomdelEnrichmentData = makeEnrichmentDataPromise({
+    readonly _copyNumberHomdelEnrichmentData = makeEnrichmentDataPromise({
```

Comment:
WithoutCytoband, for clarity and coherence

Reply:
nvm: see my comment here https://github.com/cBioPortal/cbioportal-frontend/pull/2180/files#r310723393